src/main/java/com/powsybl/openloadflow/NetworkCache.java

```diff
@@ -358,7 +370,7 @@ public Entry get(Network network, LoadFlowParameters parameters) {
             for (AcLoadFlowContext context : entry.getContexts()) {
                 AcLoadFlowResult result = context.getResult();
                 if (result != null && result.getNewtonRaphsonStatus() == NewtonRaphsonStatus.CONVERGED) {
-                    context.getParameters().setVoltageInitializer(new PreviousValueVoltageInitializer());
+                    context.getParameters().setVoltageInitializer(new PreviousValueVoltageInitializer(true));
```

Comment:
This change is wanted?

Reply:
yes, this is because in some cases, we can add bus to the main cmponent by closing a switch which was not previously been calculated so when need to fallback to flat start for theses buses